FrontEnd/pom.xml

```diff
@@ -0,0 +1,202 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
```

Comment:
Can you add a TODO to fix these details? (E.g. it isn't a demo project)
On TODO's see e.g. here:
https://engdoc.corp.google.com/eng/doc/devguide/cpp/styleguide.md?cl=head#TODO_Comments

Reply:
I don't have access but I changed the artifactId and groupId (not sure about the convention)